packages/react-inbox/src/components/Messages/index.tsx

```diff
@@ -10,77 +10,79 @@ import { useAtBottom } from "~/hooks/use-at-bottom";
 import useInbox from "~/hooks/use-inbox";
 import Header from "./Header";
 
-const Messages: React.FunctionComponent<InboxProps> = ({
-  title = "Inbox",
-  renderHeader,
-  renderMessage,
-}) => {
-  const {
-    fetchMessages,
-    markAllAsRead,
-    currentTab,
-    isLoading,
-    messages = [],
-    startCursor,
-    unreadMessageCount,
-  } = useInbox();
-  const ref = useRef<HTMLDivElement>(null);
+const Messages: React.ForwardRefExoticComponent<
```

Comment:
what makes this exotic? what are the advantages I mean? I read up on it on StackOverflow but I wasn't clear if I understand the use-case

> Also, a big difference between the two definitions seems to be, that ForwardRefExoticComponent (like all exotic components) are no function components, but actually just objects, which are treated specially when rendering them. Therefore the comment

Reply:
its just the typescript name for it.